importScripts/sql/transformations/ObjectsExportView.sql

```diff
@@ -1,3 +1,14 @@
+DROP FUNCTION IF EXISTS clean_spaces;
```

Comment:
This is copypasta from the LibraryExportView. I used the function further down in the script and the library script has not yet been run